### ISSUE TITLE
Path to role tasks

### ISSUE THREAD

**Issue by aelexor** [MSDehghan/AnsiblePlugin#8]
Hello!

At the moment, as I understand, plugin indexing of a role task which are in `role_name/tasks/task_name.yml`, but when I put of the role task to `role_name/tasks/whatever/task_name.yml` plugin stops working.

Can you added indexing the following path: `role_name/tasks/whatever/task_name.yml`?

**Comment by MSDehghan** [comment]
Hello
Currently plugin searches for `roles/[role_name]/tasks/main.yml`. Do you have `main.yml` in your tasks directory?

**Comment by aelexor** [comment]
Yes, `main.yml` exists in my directory, but `main.yml` contains include tasks, for example:
`- include: configure/main.yml`
In this way I have the following structure:
`roles/[role_name]/tasks/main.yml`
`roles/[role_name]/tasks/configure/main.yml`

**Comment by MSDehghan** [comment]
You're right. Plugin does not detect files in tasks sub-directories. I'll fix that.
Thank you

**Comment by azrdev** [comment]
generally it should be possible to apply the plugin to all yaml files, e.g. also playbooks
